src/containers/edit-tutor-profile/NotificationContainer.style.ts

```diff
@@ -0,0 +1,61 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  root: {
+    backgroundColor: 'basic.white',
+    padding: '20px',
+    borderRadius: '5px'
+  },
+  titleWithDescription: {
+    title: {
+      typography: TypographyVariantEnum.H5
+    },
+    description: {
+      typography: TypographyVariantEnum.Subtitle2,
+      color: 'basic.gray'
+    }
+  },
+  optionsContainer: {
+    padding: '30px',
+    pr: '50px'
+  },
+  options: {
+    border: '1px solid ',
+    borderColor: 'basic.gray',
```

Comment:
can use `palette` and write it in one `border` property.
What do you think?

Reply:
it`s the same, just another option

Reply:
> it`s the same, just another option

I don't see the point of creating a separate option for the border color if we can define it once in the `border` property. 
Look at the example
<img width="496" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/5ac08f00-82e3-469d-a576-b3f92be41e39">

Reply:
I will not fix it, it is just the same + we need to import pallete

Reply:
> I will not fix it, it is just the same + we need to import pallete

> I will not fix it, it is just the same + we need to import pallete

In your opinion `the same`. 
we can just write `padding` in 4 properties it is also will be the same but we make it easier in one property.
the question. that we should to avoid duplication of code ?
At the screen example how to make it easier and more cleaner in one property as I said before

If you decide leave it - please explain why it will be better to write it in several properties ? 
Advantages and disadvantages
Explaining why are you decide to leave it ?

Reply:
> I will not fix it, it is just the same + we need to import pallete

In your opinion the same.
we can just write padding in 4 properties it is also will be the same but we make it easier in one property.
the question. that we should to avoid duplication of code ?
At the screen example how to make it easier and more cleaner in one property as I said before

If you decide leave it - please explain why it will be better to write it in several properties ?
Advantages and disadvantages
Explaining why are you decide to leave it ?

Reply:
using border and borderColor more clear and easy to modify if needed than write border:`1px solid ${'basic.gray'}`  or border:`1px solid ${pallete.basic.gray}`

Reply:
> using border and borderColor more clear and easy to modify if needed than write border:`1px solid ${'basic.gray'}` or border:`1px solid ${pallete.basic.gray}`

You said: `more clear and easy to modify if needed` you didn’t give any arguments..
When you group related styles into one declaration, they become easier to understand and maintain
I don't see the point in your words